README.md

```diff
@@ -17,3 +23,44 @@ You will need to build and run the spin components. Change the cron expression i
 cd guest
 spin build --up
 ```
+
+
+## Installing the template 
+
+The template can be installed using the following comman:
+
+```bash
+spin templates install --git https://github.com/fermyon/spin-trigger-cron
+```
+
+Once the template is installed, a new project can be instantiated using:
+
+```bash
+spin new -t cron-rust hello_cron --accept-defaults
+```
+
+To run the newly created app:
+
+```
+cd hello_cron
+spin build --up
+```
+
+## Trigger Configuration
+
+The trigger type is `cron` and there are no application-level configuration options. 
+
+The following options are available to set in the [[trigger.cron]] section:
+
+| Name                  | Type             | Required? | Description |
+|-----------------------|------------------|-----------|-------------|
+| `component`           | string or table  | required  | The component to run when a queue message is received. (This is the standard Spin trigger component field.) |
+| `cron_expression`     | string           | required  | The `cron` expresison describing the interval at which the component is executed. |
```

Comment:
```suggestion
| `cron_expression`     | string           | required  | The `cron` expresison describing the schedule on which to execute the component. |
```

(because it's not necessarily a regular interval)